### DISCUSSION TITLE
Creating a Tina Learning Center

### DISCUSSION THREAD

**Discussion by perkinsjr** [tinacms/tinacms#2383] [Ideas]
I think the next step in Tina documentation is a Tina Learning Center. Where a user can interact and learn at the same time, this has become popular with a lot of projects for example:

https://nextjs.org/learn/basics/create-nextjs-app 
https://reactbricks.com/learn/welcome
https://graphcms.com/academy

Thoughts?

**Comment by jamespohalloran** [comment]
I do like the interactive element of those examples! Are you thinking this would be updating our existing concepts-docs or guide documents? or would this be a whole new set of content?

**Reply by perkinsjr** [reply]
It would be an entirely separate piece of content from our documentation and guides. It would allow us to create meaningful interactive tutorials surrounding the core concepts of using Tina from the ground up.
